src/toHaveStyleRule.js

```diff
@@ -92,41 +97,31 @@ const getDeclaration = (rule, property) =>
 const getDeclarations = (rules, property) =>
   rules.map(rule => getDeclaration(rule, property)).filter(Boolean)
 
-const normalizeModifierOption = modifier =>
-  Array.isArray(modifier) ? modifier.join('') : modifier
+/* eslint-disable prettier/prettier */
```

Comment:
nit

```js
const normalizeOptions = ({ modifier, ...options }) => {
  if (modifier) {
    return {
      ...options,
      modifier: Array.isArray(modifier) ? modifier.join('') : modifier,
    }
  }

  return options
}
```

Reply:
I'm happy to implement following your suggestion, however I usually try to avoid "if statements" when they're not strictly required and in this case I did definitely prefer a simple ternary which also allowed to take advantage of ES6 arrow function implicit return as opposed to open a block.